sql/npc_list.sql

```diff
@@ -29548,7 +29548,7 @@ INSERT INTO `npc_list` VALUES (17764581,'Nokkhi_Jinjahl','Nokkhi Jinjahl',36,3.3
 INSERT INTO `npc_list` VALUES (17764582,'Ominous_Cloud','Ominous Cloud',7,-20.632,-2.939,-40.554,16,40,40,0,0,0,2,27,0x0000920000000000000000000000000000000000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17764583,'Samigo-Pormigo','Samigo-Pormigo',34,-9.782,-5.249,-134.432,1,40,40,0,1,0,0,27,0x01000C0579107920003000406E50006000700000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17764584,'Hauh_Colphioh','Hauh Colphioh',50,-38.173,-1.250,-113.679,21,40,40,0,1,0,0,27,0x010004077710772017300E401450006000700000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17764585,'Kopua-Mobua_AMAN','Kopua-Mobua A.M.A.N.',107,-23.134,2.749,-67.284,14,40,40,0,0,0,0,27,0x0100020575106720683066406850006000700000,32,'SOA',1);
+INSERT INTO `npc_list` VALUES (17764585,'Kopua-Mobua_AMAN','Kopua-Mobua A.M.A.N.',107,-23.134,2.749,-67.284,14,40,40,0,0,0,0,27,0x0100020575106720683066406850006000700000,32,NULL,1);
```

Comment:
tbh I think it might be better to settle on TOAU for the AMANs....

Looks like [AMAN came out in TOAU](https://ffxiclopedia.fandom.com/wiki/Adventurers%27_Mutual_Aid_Network?action=history), but from this GamerEscape Wiki it looks like Kopua-Mobua was an [NPC before he got the AMAN suffix](https://ffxi.gamerescape.com/wiki/Kopua-Mobua_A.M.A.N.). I assume Kopua-Mobua in his old form no longer exists?

Reply:
The three I made Null I can put CoP on. They seemed to all be there pre-toau for mentor NPCs which are coded